nixos.nix

```diff
@@ -90,13 +90,67 @@ in
         # Create a directory in persistent storage, so we can bind
         # mount it.
         mkDirCreationSnippet = persistentStoragePath: dir:
```

Comment:
The name and the comment should to be updated to reflect the new behavior.

Reply:
Done